modules/tools/kube-downscaler/values.tpl

```diff
@@ -5,7 +5,7 @@ image:
     %{~ if excluded_namespaces != "" ~}
     - --exclude-namespaces=${excluded_namespaces}
     %{~ endif ~}
-    - --exclude-deployments=kube-downscaler,metrics-server,cluster-autoscaler-aws-cluster-autoscaler
+    - --exclude-deployments=kube-downscaler,metrics-server,cluster-autoscaler-aws-cluster-autoscaler,prometheus-kube-prometheus-stack-prometheus,kube-prometheus-stack-kube-state-metrics
```

Comment:
I want the downscaler exceptions to be applied as annotations on the deployments instead of added to the downscaler configuration.